### ISSUE TITLE
Exercise: OA Literature search

### ISSUE THREAD

**Issue by johav** [OpenScienceMOOC/Module-6-Open-Access-to-Research-Papers#2]
Likeminded initiatives already developed exercises that we could include, such as
- https://github.com/OpenKnowledgeMaps/learning-materials/blob/master/mozilla-study-group-lesson.md (CC-BY-SA Open Knowledge Maps)

**Comment by Protohedgehog** [comment]
Yes, absolutey! I think we should just be able to re-use that given the licensing?

**Comment by tosteiner** [comment]
The ubiquitous FOSTER project also has a variety of input on that matter:

* https://www.fosteropenscience.eu/learning/open-access-publishing/
* https://www.fosteropenscience.eu/learning/open-access-to-publications-in-horizon-2020/

and re: https://github.com/OpenScienceMOOC/Module-6-Open-Access-to-Research-Papers/issues/2#issuecomment-426762657 - yes, BY SA means `Attribution, Share Alike`, so we need to attribute the original author and share our exercise under the same license as theirs

**Comment by Protohedgehog** [comment]
Yes, @tosteiner, we'll definitely re-use FOSTER's material (e.g., videos) where possible! No point re-inventing the wheel :)

**Comment by tosteiner** [comment]
If need be, we could also ask people to file through collections on ScienceOpen... see e.g. [our own collection with OA filter applied](https://www.scienceopen.com/search#('v'~3_'id'~''_'isExactMatch'~true_'context'~('collection'~('id'~'6db2657e-c46d-497f-ba37-b0079802d907'_'kind'~0)_'kind'~11)_'kind'~77_'order'~3_'orderLowestFirst'~false_'query'~'open%20access'_'filters'~!*)) 😎

**Comment by Protohedgehog** [comment]
That's a nice idea. We could probably ask Stephanie to record a demo video for that too to help out.

**Comment by Protohedgehog** [comment]
Hi folks. Just picking up on this. Shall we go for the OK Maps task? I can get in touch with Peter and ask if we can adapt it for this.
